charts/stable/gollum/values.yaml

```diff
@@ -0,0 +1,106 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- gollum upstream image
+  repository: gollumorg/gollum
+  tag: latest
+  pullPolicy: IfNotPresent
+
+# -- For daemonset see persistence values, RWX will be required
+controllerType: statefulset
+# -- Start command
+command: ["gollum", "/wiki", "--config", "/config/config.rb", "--h1-title"]
```

Comment:
This should be moved outside of the values.yaml to ensure it does not infer it should be changed.